### ISSUE TITLE
i18n procedures not great

### ISSUE THREAD

**Issue by dascritch** [mmuman/RadioTimer#2]
Get a look at <https://github.com/dascritch/__> ;)

**Comment by mmuman** [comment]
Yeah I know, I heard there was an implementation of gettext as well, but I didn't get the time to look at it.

**Comment by mmuman** [comment]
I just switched to using the lang HTML attribute at least, it's less ugly than class="xx".
